### ISSUE TITLE
Regional settings

### ISSUE THREAD

**Issue by dumoulins** [BlueQuartzSoftware/DREAM3D#569]
Hello,

We experience issues with the decimal separator (local/regional settings), with the StatsGenerator filter and the dynamic tables, so far. For a the user who has the regional settings such that the decimal separator is a comma:
For the StatsGenerator filter -> US/UK format (i.e. dots instead of comma) appears in the GUI (for example mu and sigma) when either opening an existing json file or when creating a new StatsGenerator filter (for the latter, I guess this means the default values are set in US/UK format without accounting for the regional settings)
We experience the same problem with the filters we have made that use dynamic tables.

Regards
Stéphane

**Comment by imikejackson** [comment]
Could you please try the latest nightly builds located at http://dream3d.bluequartz.net/binaries/experimental/  and then look for the current date. I believe that we have updated those widgets in the past few weeks.

**Comment by dumoulins** [comment]
We did try with the nightly build from 10.08.2016 (DREAM3D-6.3.72.b431097-Win64) but the problem is still there with StatsGenerator (we haven't tried with the dynamic tables).
When we create a new pipeline with the StatsGenerator, then Mu and Sigma appear with dots as decimal separators while the "Bin Step Size" has a comma (the local settings are based on the comma).

Furthermore, we experienced DREAM3D to crash when Sigma is set to 0, see the attached screen shot. 

![snipimage](https://cloud.githubusercontent.com/assets/18528094/17581117/bb70594e-5fa3-11e6-9054-5a7ce9911dd1.JPG)
